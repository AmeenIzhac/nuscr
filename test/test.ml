let test_files = [ "/examples/parser/first.scr"
                 ; "/examples/parser/GProtocol.scr"
                 ; "/examples/parser/GMessage.scr"
                 ]

(* test the parser *)
let () =
  try
    let pwd =  Sys.getenv "PWD" in
    let files = List.map (fun x -> (pwd ^ x)) test_files in
    let _ = List.map Nuscrlib.process_file files in
    print_endline "Ok"
  with
  | e -> "Not ok\n" ^ Printexc.to_string e |> print_endline
