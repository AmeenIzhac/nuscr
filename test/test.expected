Found 75 tests.
Ok
